code/modules/mob/living/silicon/robot/sprites/_sprite_datum.dm

```diff
@@ -1,7 +1,28 @@
+#define ROBOT_HAS_SPEED_SPRITE 0x1	//Ex:	/obj/item/borg/combat/mobility Replaces old has_speed_sprite
```

Comment:
defines need to go to the _defines folder or need to get undefined

move those best to the defines section